### DISCUSSION TITLE
ra8d1 usb fs device

### DISCUSSION THREAD

**Discussion by wugensheng-code** [hathach/tinyusb#2886] [Q&A]
I have an evaluation board for Renesas RA8d1, and I am trying to run tinyusb on a new RTOS. Previously, this board had run the rt thread tinyusb CDC program, which proves that there were no hardware issues. Currently, I am encountering some problems:
Unable to process the setup package sent by the computer, no Control transfer stage changes were generated. I printed out the interrupt type in the interrupt service function, and the log is as follows. Can anyone provide suggestions
![tinyusb](https://github.com/user-attachments/assets/05f25cfa-0ed8-4289-a476-d5acfd034ec7)

**Comment by hathach** [comment]
Ra8 is not supported yet. Rt may make their own modifications
